web_src/css/repo/home-file-list.css

```diff
@@ -22,18 +33,32 @@
   display: contents;
 }
 
-#repo-files-table .repo-file-item:hover > .repo-file-cell {
-  background: var(--color-hover);
+#repo-files-table .repo-file-item:hover > .repo-file-cell,
+#repo-files-table .parent-link:hover {
+  background: var(--color-hover-opaque);
 }
 
 #repo-files-table .repo-file-line,
 #repo-files-table .repo-file-cell {
-  border-top: 1px solid var(--color-light-border);
+  border-top: 1px solid var(--color-secondary-light-1);
   padding: 8px 10px;
 }
 
 #repo-files-table .repo-file-line:first-child {
   border-top: none;
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
+}
+
+#repo-files-table .repo-file-item:nth-child(2) .repo-file-cell {
+  border-top-color: var(--color-secondary);
+}
```

Comment:
Why `nth-child(2)`?

```
line
----
line parent
----
item1
---- < only this one?
item2
----
item3
```

Reply:
OK, I am sure the `nth-child(2)` is not right, because there could be 1 or 2 "lines" above.

Reply:
Hmm yes, it needs a `+` selector.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/32835/commits/cc3c2d510a6cc9c13bdabbc630d3bb545e9e076b.

Reply:
Ok I guess same color is better as we don't do this in other tables, so I reverted it to same color in https://github.com/go-gitea/gitea/pull/32835/commits/dc821e32c2a94f865b59708a3b1eca0c51ff7739.